local_ssd_user_data_script/README.md

```diff
@@ -0,0 +1,18 @@
+# User Data Startup Script for local ssd support
+
+This folder contains a template of a user data script to support local ssd on Ocean spark platform.
+
+## Usage
+On Spot Ocean Cluster  
+Add the bash script on the VNGs used for spark executors.  
+Replace the value of the variable EKS_CLUSTER_NAME by the name of the cluster.
+
+
+## Explanation
+This script will create a folder "/data" for all nodes of the VNG.  
+If the node is an instance using NVME volume (SSD), it will mount this volume into the above folder "/data".  
+Otherwise, the folder will just use current volume.  
+
+Ocean Spark will mount this folder as a volume for the spark application executors.  
+
+Currently, it's enabled only for spark 3 and EKS.
```

Comment:
```suggestion
This script will create a folder `/data` on the filesystem of all nodes in the VNG.  
If the EC2 instance underlying the node uses NVMe volumes (like `m5d`, `r5d`, or `i3` families), the script will mount these volumes into the above folder `/data`.  
Otherwise, the `/data` folder will just be transparently created on the instance boot disk.  

Ocean Spark will mount this folder as a volume in all Spark executor pods.  

Currently, local SSD support is enabled for Spark 3 and AWS only.
```